### PR TITLE
Update to v0.14.0-rc3

### PR DESCRIPTION
**Pull request by paluh**
I've incorporated PR #65 by @kl0tl here.

Backlinking to purescript/purescript#3942

### CODE REVIEW

**Review by JordanMartinez — APPROVED** [review]
This will be ready to merge once `st` is merged.

**Review by hdgarrood — APPROVED** [review]

**Pull request merged**
